src/context/UserContext/UserContext.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { useState, createContext, useContext } from 'react';
+
+export const defaultValue = {
+    isAttestant: false,
+    setIsAttestant: () => undefined,
+};
+type UserContext = {
+    isAttestant: boolean;
+    setIsAttestant: React.Dispatch<React.SetStateAction<UserContext>>;
+};
+const UserContext = createContext<UserContext>(defaultValue);
+
+export const UserProvider = (props: { children: React.ReactNode }) => {
+    const [value, setValue] = useState<UserContext>(defaultValue);
+    return (
+        <UserContext.Provider
+            value={{
+                isAttestant: value.isAttestant,
+                setIsAttestant: setValue,
+            }}
+        >
+            {props.children}
+        </UserContext.Provider>
+    );
+};
+
+const useUserContext = () => useContext(UserContext);
+export default useUserContext;
```

Comment:
Jeg syns det ser rart ut med `import useUserContext from '..../UserContext'`. Filnavnet ser ut som at det er en React-komponent som eksporteres som default.
Det er også kun denne filen nå, så det føles litt redundant å ha den inni ei mappe.

Forslag: flytt og rename til `src/context/userContext.tsx` og eksporter slik:

```suggestion
export const useUserContext = () => useContext(UserContext);
```